cmd/containerd-shim-runhcs-v1/pod.go

```
@@ -176,7 +176,7 @@ func createPod(ctx context.Context, events publisher, req *task.CreateTaskReques
 		p.sandboxTask = newWcowPodSandboxTask(ctx, events, req.ID, req.Bundle, parent, nsid)
 		// Publish the created event. We only do this for a fake WCOW task. A
 		// HCS Task will event itself based on actual process lifetime.
-		events.publishEvent(
+		_ = events.publishEvent(
```

Think we should also return the error here.